### DISCUSSION TITLE
Is there a chrome extension powered by this repository?

### DISCUSSION THREAD

**Discussion by hirehamir** [Automattic/harper#322] [Q&A] [answered]
These searches have no results at the time of this writing.
https://github.com/Automattic/harper/issues?q=chome%20extension
https://github.com/Automattic/harper/discussions?discussions_q=chome+extension

**Comment by elijah-potter** [comment, marked as answer]
Hey Hamir,

The short answer is: not yet.

We hope to make a Chrome extension _eventually,_ but not soon. There are simply higher-priority things on the roadmap. I believe someone made an attempt a while ago, but haven't heard from them since.

That said, we are about to create the first release of `harper.js`, which should make it easy to make such an extension if you (or someone reading this) has prior experience with the domain.

**Reply by hirehamir** [reply]
Oh, nice. Are you planning to have full TypeScript support? It can catch a lot of issues during development.

**Reply by elijah-potter** [reply]
> Oh, nice. Are you planning to have full TypeScript support? It can catch a lot of issues during development.

We are! It should be out next week. I'll ping you here when you can take a full look.

If you're impatient, you can read the source in the Harper repo under `packages/harper.js`. :)

**Reply by elijah-potter** [reply]
It's out! [Here's some info about a potential Chrome extension](https://writewithharper.com/docs/faq#Is-There-a-Chrome-or-Firefox-Extension-) and [here's the documentation on `harper.js`](https://writewithharper.com/docs/harperjs/introduction).

**Reply by elijah-potter** [reply]
[An update on the topic](https://elijahpotter.dev/articles/status_of_the_harper_chrome_extension),
